simplq/src/components/pages/Join/Form.jsx

```diff
@@ -77,7 +82,17 @@ export function JoinQueueForm(props) {
                 inputStyle={{
                   width: '100%',
                 }}
-                isValid={() => (invalidContact ? 'Phone number is not valid' : true)}
+                isValid={(inputNumber, country) => {
+                  var arr=[];
+                  arr = inputNumber.split(country.dialCode);
+                  if(arr[1].length!=10){
+                    setInvalidContactLength(true)
```

Comment:
Make it !== here

Reply:
Also, arr[1].length will cause the app to crash if the user enters one digit and then deletes it. The code would get undefined in arr and would crash at arr[1].length, so that case should be handled.

Reply:
The logic used here would work for Indian phone numbers but not other countries. I tried for a Hong Kong number and I can't get a valid number. This isn't an essential thing and can be looked into later if it's too much work.

Reply:
Okkk , Than lets keep it as it is